src/main/java/io/kuzzle/sdk/Options/ListOptions.java

```diff
@@ -0,0 +1,47 @@
+package io.kuzzle.sdk.Options;
+
+import java.util.concurrent.ConcurrentHashMap;
+
+public class ListOptions {
+  private Integer from;
+  private Integer size;
+  /**
+   * Constructor
+   */
+  public ListOptions() {}
+
+  /**
+   * Copy constructor
+   *
+   * @param options
+   */
+  public ListOptions(ListOptions options) {
+    this.from = options.getFrom();
+    this.size = options.getSize();
+  }
+
+  public Integer getFrom() {
+    return from;
```

Comment:
What is returned when the uninitialized value is returned? The request contain a number or null?

Reply:
If its uninitialized it returns null